index.yaml

```diff
@@ -7,9 +7,9 @@ entries:
       catalog.cattle.io/release-name: ambassador
     apiVersion: v1
     appVersion: 1.13.8
-    created: "2021-06-25T13:41:52.528853-07:00"
+    created: "2021-08-16T16:58:12.196947-07:00"
     description: A Helm chart for Datawire Ambassador
-    digest: 395a8bfddf57116dabaa5f96ef027e05094b25c720947223f9167843cf4e7319
```

Comment:
digests do not get updated by Helm if the chart version remains the same since Helm does not update the index.yaml for existing entries.